packages/@aws-cdk/cloud-assembly-schema/lib/assembly-manifest.ts

```diff
@@ -0,0 +1,340 @@
+interface BaseAssetMetadataEntry {
```

Comment:
Rename to `AssetMetadataEntry` maybe?

Should have a tsdoc that describes what asset metadata and asset metadata entry are.

Reply:
Well its not exported... :)

But yeah i'll add it. And i like the renaming as well.

Reply:
> Well its not exported

Unlike other modules, the main audience for the `cloud-assembly-schema` module is going to be (at least for the time being) the CDK team. Adding documentation makes it clearer to the author (modifying the schema) and the reviewer whether the change to this class is correct.